File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        PriorityQueue<int, int> minHeap = new();

        foreach (int i in nums) {
            minHeap.Enqueue(i, i);
        }

        while(minHeap.Count > k) {
            minHeap.Dequeue();
        }

        return minHeap.Peek();
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-2.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        p1 = 0
        dup_array = []

        while p1 <= len(nums) - 1:
            if(nums[p1] in dup_array):
                return True
            else:
                dup_array.append(nums[p1])
                p1 += 1
        return False
         